spec/index.html

```diff
@@ -2157,8 +2159,36 @@ <h5>The <code>&lt;<dfn id="the-feature-element">feature</dfn>&gt;</code> element
         </dl>
 
         <p>
-          A <a href="#the-feature-element"><code>feature</code></a> element represents a geographic feature. A <a href="#the-feature-element"><code>feature</code></a> element has an optional <a href="#the-properties-element"><code>properties</code></a> child element, and a required child
-          <a href="#the-geometry-element"><code>geometry</code></a> element.
+          A <a href="#the-feature-element"><code>feature</code></a> element represents a map feature. 
+        </p>
+        </section>
+        <section>
+        <h5>The <code>&lt;<dfn id="the-featurecaption-element">featurecaption</dfn>&gt;</code> element</h5>
+        <dl class="def">
+          <dt><a href="https://html.spec.whatwg.org/multipage/dom.html#concept-element-categories">Categories</a>:</dt>
+          <dd>Feature data.</dd>
+          <dt><a href="https://html.spec.whatwg.org/multipage/dom.html#concept-element-contexts">Contexts in which this element can be used</a>:</dt>
+          <dd>Child of the <a href="#the-body-element"><code>feature</code></a> element.</dd>
```

Comment:
This is referencing `feature` but is linking to `#the-body-element`, change to `#the-feature-element`.

Reply:
face-palm.gif